app/controllers/hackdays_controller.rb

```diff
@@ -0,0 +1,29 @@
+class HackdaysController < ApplicationController
+
+  def index
+    @hackday = Hackday.new
+    @hackdays = Hackday.all
+  end
+
+  def show
+    @hackday = Hackday.find(params[:id])
```

Comment:
should always use find_by_id rather than find.

find throws an exception that sucks